src/Sarif/Schemata/sarif-schema.json

```diff
@@ -2679,6 +2728,49 @@
           }
         },
 
+        "language": {
+          "description": "The language of the the localized strings defined in this component (expressed as an ISO 649 two-letter lowercase culture code) and region (expressed as an ISO 3166 two-letter uppercase subculture code associated with a country or region).",
+          "type": "string"
+        },
+
+        "contents": {
+          "description": "The kinds of data contained in this object.",
+          "type": "array",
+          "uniqueItems": true,
+          "default": [ "localizedData", "nonLocalizedData" ],
+          "items": {
+            "enum": [
+              "localizedData",
+              "nonLocalizedData"
+            ]
+          }
+        },
+
+        "isComprehensive": {
+          "description": "true if this object contains a complete definition of the localizable and/or non-localizable data for this component.",
```

Comment:
>true [](start = 26, length = 4)

'Specifies whether this object'
add to end 'as to opposed to including only data that is relevant to the results persisted to this log file.'